Source/WebCore/Modules/applepay/ApplePayError.h

```diff
@@ -38,9 +38,14 @@ namespace WebCore {
 
 class ApplePayError final : public RefCounted<ApplePayError> {
 public:
-    static Ref<ApplePayError> create(ApplePayErrorCode code, std::optional<ApplePayErrorContactField> contactField, const String& message)
+
+    enum class Domain : uint8_t {
```

Comment:
I might have asked this in the past, but can we make this a `bool` backed type for tighter packing?

Reply:
We did resolve this one earlier :D